dask/array/__init__.py

```diff
@@ -13,5 +13,5 @@
 from .percentile import percentile
 with ignoring(ImportError):
     from .reductions import nanprod
-from . import random, linalg, ghost
+from . import random, linalg, ghost, creation
```

Comment:
We probably don't want `creation` in the main namespace but do want arange and linspace.

Recommended usage is 

    import dask.array as da
    import numpy as np

Ideally da mirrors np as much as possible